app/src/utils/upload-file.ts

```diff
@@ -1,54 +1,142 @@
 import api from '@/api';
 import { emitter, Events } from '@/events';
 import { i18n } from '@/lang';
+import { useServerStore } from '@/stores/server';
 import { notify } from '@/utils/notify';
 import type { AxiosProgressEvent } from 'axios';
 import { unexpectedError } from './unexpected-error';
+import * as tus from 'tus-js-client';
```

Comment:
Can we import the individually used parts? That way vite has a chance at tree-shaking 🤞🏻